### DISCUSSION TITLE
Does your newest update apply to older versions of Minecraft such as 1.12.2?

### DISCUSSION THREAD

**Discussion by ProblemsSender** [TownyAdvanced/Towny#5438] [General]
If not, do you plan to do so in the future? Because some server owners maybe choose to stay on these legacies version... Or at least I'd expected the plugin to be free from bugs if there's one. Thanks

**Comment by LlmDl** [comment]
Hi @ProblemsSender every once in a while I do a backport of Towny's latest release to 1.12.2. It takes a bit of time to do it right. If you are on 1.12.2. you can use [0.97.0.0 for 1.12.2](https://github.com/TownyAdvanced/Towny/releases/tag/0.97.0.0for1.12.2). 

I understand you're wanting a more up to date version of Towny for your 1.12.2 server. It might interest you that I am trying to get to 100 [sponsors ](https://github.com/sponsors/LlmDl) here on github.
